code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -59,7 +59,7 @@
 
 /obj/machinery/airalarm
 	name = "air alarm"
-	desc = "A machine that monitors atmosphere levels and alerts if the area is dangerous."
+	desc = "A machine that monitors atmosphere levels. Goes off if the area is dangerous."
```

Comment:
Old description sounded better

Reply:
that's what it is standard, I didn't change it

Reply:
This entire file seems to have a lot of dirty changes.